suite-native/atoms/src/Sheet/BottomSheet.tsx

```diff
@@ -39,6 +40,7 @@ const sheetWithOverlayStyle = prepareNativeStyle(_ => ({
 
 export const BottomSheet = ({
     isVisible,
+    isCloseDisplayed = true,
```

Comment:
No need to have extra property, just use `onClose` and check if it's undefined or not